app/controllers/users.py

```diff
@@ -1142,8 +1144,21 @@ def post(self, user_id):
             log_activity('User', status='Success',
                          operation='Disable',
                          description='Disabled user Successfully',
-                         a_project_id=project.id,
-                         a_cluster_id=project.cluster_id)
+                         )
+            #send email
+            html_layout = render_template(
+                'user/user_disable_enable.html',
+                email=user.email,
+                name=user.name,
+                status='disabled')
+            send_email(
+                'lanternnassi@gmail.com',
```

Comment:
This is hard coded too
```suggestion
                user.email,
```